sys/Kconfig.stdio

```diff
@@ -11,6 +11,8 @@ menu "Standard Input/Output (STDIO)"
 choice STDIO_IMPLEMENTATION
     bool "STDIO implementation"
     default MODULE_STDIO_NATIVE if CPU_ARCH_NATIVE
+    default MODULE_STDIO_CDC_ACM if TEST_KCONFIG && HAS_HIGHLEVEL_STDIO && !MODULE_TINYUSB_DEVICE
```

Comment:
Checking for `TEST_KCONFIG` is not required, as it is inherited from the menu dependency.
I'm wondering whether now, that `stdio_cdc_acm` is also implemented by tinyUSB, `MODULE_STDIO_CDC_ACM` shouldn't be moved to this file? It could even be made a `menuconfig` with a `choice` to select the backend (see my other comment on this).